spec/controllers/api/current/analytics/user_analytics_controller_spec.rb

```diff
@@ -1,45 +1,85 @@
 require 'rails_helper'
 
 RSpec.describe Api::Current::Analytics::UserAnalyticsController, type: :controller do
-  render_views
-
   let!(:request_user) { create(:user) }
   let!(:request_facility) { create(:facility, facility_group: request_user.facility.facility_group) }
 
-  let!(:patients) { create_list :patient, 10, registration_facility: request_facility, registration_user: request_user }
-
   before :each do
     request.env['HTTP_X_USER_ID'] = request_user.id
     request.env['HTTP_X_FACILITY_ID'] = request_facility.id
     request.env['HTTP_AUTHORIZATION'] = "Bearer #{request_user.access_token}"
   end
 
-  describe 'GET: send data from server to device;' do
-    it 'gets data for 4 weeks as a hashmap' do
-      get :show, format: :json
+  describe '#show' do
+    context 'json_api' do
+      describe 'facility has no patients registered' do
+        it 'gets data for as a hashmap' do
```

Comment:
This asserts for `nil` and not a hashmap

Reply:
Fixed